front/src/service.ts

```diff
@@ -420,4 +421,71 @@ export class ScratchyService {
         this.messageCache.delete(messageID);
         return response.data;
     }
-}
\ No newline at end of file
+
+/* Call server with Websocket */
+
+     /**
+     *
+     * @async
+     * @augments ScratchyService
+     * @param {Room} room - affected room
+     * @returns {Promise<undefined>} -
+     */
+    async joinRoomWebsocket(room: Room): Promise<void> {
+        socket.on("join", (socket) => {  
```

Comment:
No need to 'on' just emit, the server is the one listening.

Reply:
Done here  [66972bc](https://github.com/Draeggon/scratchy/pull/2/commits/66972bc3403d4e41c5a35ee175b805c6b02e7d60)